0-bootstrap/modules/jenkins-agent/README.md

```diff
@@ -116,8 +116,8 @@ Error: failed pre-requisites: missing permission on "billingAccounts/aaaaaa-bbbb
 
 -   [gcloud sdk](https://cloud.google.com/sdk/install) >= 206.0.0
 -   [Terraform](https://www.terraform.io/downloads.html) >= 0.12.6
--   [terraform-provider-google] plugin 2.1.x
--   [terraform-provider-google-beta] plugin 2.1.x
+-   [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin 2.1.x
```

Comment:
Not sure why we are calling out `plugin 2.1.x.

Reply:
I originally copied this from the bootstrap module: https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/README.md#software

But I can remove it now in my next push